src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -375,9 +376,14 @@ const MapView: React.FC<MapViewProps> = ({
 
             // TODO: update this to if the tree was planted within the past three years
             // If the tree has not been updated within the past three years, use youngTreeIcon
-            const updatedDate = feature.getProperty('updated_at');
-            if (updatedDate < breakpointDate) {
+            const plantedDate = feature.getProperty('plantingDate');
+            const adopted = !!feature.getProperty('adopterId');
+            if (adopted) {
+              icon = adoptedTreeIcon;
+            } else if (!!plantedDate && plantedDate > breakpointDate) {
```

Comment:
❓ What's the difference between putting `!!` here versus before `feature.getProperty('plantingDate')`?

Reply:
This notation (!!) casts the returned value to a boolean based on whether or not it is truthy. Because we need to know both whether or not a planting date exists and what the value of that date is, we have to assign the actual value to a variable first